frontend/src/component/Sidebar/WorkspaceListItem.jsx

```diff
@@ -64,7 +67,10 @@ const WorkspaceListItem = props => {
       <AnimateHeight duration={500} height={props.isOpenInSidebar ? 'auto' : 0}>
         <ul className='sidebar__content__navigation__workspace__item__submenu'>
           {props.allowedAppList.map(aa =>
-            <li key={aa.slug}>
+            <li
+              data-cy={`sidebarSubdropdown-${aa.slug}`}
```

Comment:
we used _ as a separator on every other data-cy, we should stick to it

Reply:
what means `aa` in `${aa.slug}`. This is a bad variable name. allowedApp (if it is this) must be used in this type of case. Please avoid to save chars this way.

Reply:
done (both)